src/components/FloatingAction/index.test.tsx

```diff
@@ -5,7 +5,7 @@ import FloatingAction from '.';
 
 describe('FloatingAction component', () => {
   it('renders component (set all props)', () => {
-    render(<FloatingAction Icon={() => null} label="delete the internet" />);
+    render(<FloatingAction Icon={() => <></>} label="delete the internet" />);
```

Comment:
Is there a reason we're passing a fragment here now?

Reply:
it's expecting an `Element`, the alternative is importing any Icon and pass it there, but seeing that it was passing a null, a fragment made sense.

Reply:
Hmm... my sort of gut feeling is that when we create a component, specifying that it returns a `JSX.Element` makes sense. However, specifying a prop that accepts a JSX.Element seems like a bad idea unless we explicitly know we want that.

The change I'd prefer I think (happy to chat about it) is to change this back to null, and in `src/components/Icon/index.tsx`, change line `12` to say 

```
type Icon = React.ComponentType<IconProps>
```

Reply:
i agree with that, updated.